modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -178,11 +178,6 @@ public void processOpts() {
         // map to Dot instead of Period
         specialCharReplacements.put(".", "Dot");
 
-        if (StringUtils.isEmpty(System.getenv("PYTHON_POST_PROCESS_FILE"))) {
```

Comment:
This was already available in `AbstractPythonCodegen` which this class inherits from.